### PR TITLE
fix: balance transfers

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
Fix balance transfers. Currently, you can't make a bulk balance transfer set to `0`. This fixes FE bug and adds a try/except statement around the disbursement process for robustness.

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

File: app/client/components/transferAccount/StandardTransferAccountList.tsx [hunk 1]
```diff
@@ -277,7 +277,9 @@ class StandardTransferAccountList extends React.Component<
     const hasSelected =
       allSelected || (include_accounts && include_accounts.length > 0);
 
-    let numberSet = typeof amount === "number" && amount !== 0;
+    let numberSet =
+      typeof amount === "number" &&
+      (this.state.transferType === "BALANCE" ? amount >= 0 : amount !== 0);
```
Comment (tristanhcole) [inline app/client/components/transferAccount/StandardTransferAccountList.tsx:282]: FE bug

File: app/server/api/disbursement_api.py [hunk 2]
```diff
@@ -31,40 +31,43 @@ def make_transfers(disbursement_id, auto_resolve=False):
...
-        disbursement.credit_transfers.append(transfer)
-        if auto_resolve and disbursement.state == 'APPROVED':
-            transfer.approvers = disbursement.approvers
-            transfer.add_approver_and_resolve_as_completed()
+            disbursement.credit_transfers.append(transfer)
+            if auto_resolve and disbursement.state == 'APPROVED':
+                transfer.approvers = disbursement.approvers
+                transfer.add_approver_and_resolve_as_completed()
+        except Exception as e:
+            disbursement.errors = disbursement.errors + [str(ta) +': ' + str(e)]
```
Comment (tristanhcole) [inline app/server/api/disbursement_api.py:70]: Adds an exception catch

File: app/server/utils/credit_transfer.py [hunk 3]
```diff
@@ -393,7 +393,7 @@ def make_target_balance_transfer(target_balance,
     if target_user.transfer_account is None:
         raise TransferAccountNotFoundError('Transfer account not found')
 
-    transfer_amount = target_balance - target_user.transfer_account.balance
+    transfer_amount = Decimal(target_balance) - target_user.transfer_account.balance
```
Comment (tristanhcole) [inline app/server/utils/credit_transfer.py:396]: Fixes backend bug

**Comment by tristanhcole** [comment]
[Zero balances](https://trello.com/c/NJfdEN9Q/860-zero-balances)

**Review by michielderoos — APPROVED** [review]
This works well and looks good to me!

**Pull request merged**